src/labels_transfer/api/common_arguments_2.yaml

```diff
@@ -0,0 +1,151 @@
+          example: X_scvi
+
+    - name: Reference dataset arguments
+      arguments:
+        - name: "--reference"
+          type: "file"
+          required: false
+          description: "The reference data to train classifiers on."
+          example: reference.h5mu
+          info:
+            label: "Reference"
+            file_format:
+              type: h5mu
+              mod:
+                rna:
+                  description: "Modality in AnnData format containing RNA data."
+                  required: true
+                  slots:
+                    X:
+                      type: double
+                      name: features
+                      required: false
+                      description: |
+                        The expression data to use for the classifier's training, if `--input_obsm_features` argument is not provided.
+                    obsm:
+                      - type: "double"
+                        name: "features"
+                        example: X_scvi
+                        description: |
+                          The embedding to use for the classifier's training. Override using the `--reference_obsm_features` argument.
+                          Make sure that embedding was obtained in the same way as the query embedding (e.g. by the same model or preprocessing).
+                        required: true
+                    obs:
+                      - type: "string"
+                        name: "targets"
+                        multiple: true
+                        example: [ ann_level_1, ann_level_2, ann_level_3, ann_level_4, ann_level_5, ann_finest_level ]
+                        description: "The target labels to transfer. Override using the `--reference_obs_targets` argument."
+                        required: true
+        - name: "--reference_obsm_features"
+          type: string
+          required: false
+          description: |
+            The `.obsm` key of the embedding to use for the classifier's training. If not provided, the `.X` slot will be used instead.
+            Make sure that embedding was obtained in the same way as the query embedding (e.g. by the same model or preprocessing).
+          example: X_scvi
+        - name: "--reference_obs_targets"
+          type: string
+          default: [ ann_level_1, ann_level_2, ann_level_3, ann_level_4, ann_level_5, ann_finest_level ]
+          required: false
+          multiple: true
+          description: The `.obs` key(s) of the target labels to tranfer.
+
+    - name: Outputs
+      arguments:
+        - name: "--output"
+          type: "file"
+          direction: output
+          required: true
+          description: "The query data in .h5mu format with predicted labels transfered from the reference."
+          info:
+            label: "Output data"
+            file_format:
+              type: h5mu
```

Comment:
Files are almost identical up to this point. For a better interpretation by users/developers, I suggest naming them more intuitively. 
**Option 1**
- `common_arguments_external`
- `common_arguments_internal` or `common_arguments_pipeline`
**Option 2**
- `common_arguments_h5ad_reference`
- `common_arguments_h5mu_reference`

What do you think?

Reply:
An alternative would be to generalize code to only have one file with common arguments. The initial idea of this file was to reduce repetition. Now we have 2 almost identical files with common arguments